PWGCF/Tasks/correlations.cxx

```diff
@@ -416,6 +420,64 @@ struct CorrelationTask {
   }
   PROCESS_SWITCH(CorrelationTask, processMixedDerived, "Process mixed events on derived data", false);
 
+  BinningType pairBinning{{axisVertex, axisMultiplicity}, true}; // true is for 'ignore overflows' (true by default)
+  SameKindPair<aodCollisions, aodTracks, BinningType> pair{pairBinning, cfgNoMixedEvents, -1};
```

Comment:
I would comment the `-1` in the same way you commented the `true` in the previous line